client/components/Interfaces.tsx

```diff
@@ -0,0 +1,34 @@
+export interface ProjectQuery {
```

Comment:
This entire file can be converted from `.tsx` to a type definition file (`.d.ts`) and moved to the @types directory